### DISCUSSION TITLE
How to call API through code if API have inputs

### DISCUSSION THREAD

**Discussion by gaurikalp** [EnsembleUI/ensemble#91] [Q&A] [answered]
Without the use of Action how to call API through code if API have inputs
 ensemble.invokeAPI('createToDoError');
What is the syntax to pass input?
API:
  createToDoError:
     inputs:
       - name

**Comment by amin-nas** [comment, marked as answer]
Hi @gaurikalp - I added a new example (# 5) here:
https://studio.ensembleui.com/app/e24402cb-75e2-404c-866c-29e6c3dd7992/screen/e546b0d8-3220-4217-bd5c-181118154073

```
ensemble.invokeAPI('createToDo', { 'name': 'API call from code' } );
```
